ykrt/src/trace/mod.rs

```diff
@@ -18,8 +18,9 @@ pub(crate) mod hwt;
 
 pub(crate) use errors::InvalidTraceError;
 
-/// A tracer is an object which can start / stop collecting traces. It may have its own
-/// configuration, but that is dependent on the concrete tracer itself.
+/// A `Tracer` is a front-end to an underlying tracer approach (e.g. hardware or software tracing).
+/// That backend may have its own configuration options, which is why `Tracer` does not have a
```

Comment:
Mostly fine, but if you want to refer to "that backend", wouldn't it be better to have used the term "backend" prior?

I know that you are using "underlying tracing approach" and "backend" as synonyms, but maybe unified terminology would be clearer?

Reply:
Good point. Fixed in 0cd2907.